apps/admin/backend/src/util/export_file.ts

```diff
@@ -16,9 +16,17 @@ export function exportFile({
 }): Promise<ExportDataResult> {
   const exporter = new Exporter({
     allowedExportPatterns: ADMIN_ALLOWED_EXPORT_PATTERNS,
-    /* we're not using `exportDataToUsbDrive`, so a placeholder `getUsbDrives` is fine */
-    /* c8 ignore next */
-    getUsbDrives: () => Promise.resolve([]),
+    /* We're not using `exportDataToUsbDrive` her, so a mock `usbDrive` is OK */
+    /* c8 ignore start */
+    usbDrive: {
+      status: () =>
+        Promise.resolve({
+          status: 'no_drive',
+        }),
+      eject: () => Promise.resolve(),
+      format: () => Promise.resolve(),
+    },
+    /* c8 ignore stop */
```

Comment:
I think that this workaround is evidence that `Exporter` would be better off as a couple of functions rather than a class with state and multiple methods. I didn't want to get involved in that refactor here, though.